07_deploy_masters.sh

```diff
@@ -7,7 +7,7 @@ source common.sh
 # Note This logic will likely run in a container (on the bootstrap VM)
 # for the final solution, but for now we'll prototype the workflow here
 export OS_TOKEN=fake-token
-export OS_URL=http://api.ostest.test.metalkube.org:6385/
+export OS_URL=http://localhost:6385/
 
 wait_for_json ironic \
     "${OS_URL}/v1/nodes" \
```

Comment:
Not specific to this patch but I noticed the wait_for_ironic thing doesn't work.

I noticed this because the default makefile target is still missing ironic, so the containers weren't ruinning:

```
[shardy@tripleodev2 dev-scripts]$ curl -g -X GET http://localhost:6385//v1/nodes '-H Accept: application/json -H Content-Type: application/json -H User-Agent: wait-for-json -H X-Auth-Token: fake-token'
curl: (7) Failed connect to localhost:6385; Connection refused
[shardy@tripleodev2 dev-scripts]$ curl -g -X GET http://localhost:6385//v1/nodes '-H Accept: application/json -H Content-Type: application/json -H User-Agent: wait-for-json -H X-Auth-Token: fake-token' | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed connect to localhost:6385; Connection refused
[shardy@tripleodev2 dev-scripts]$ echo $?
0
```

```
Waiting for ironic to respond++ date +%s
+ start_time=1550658918
+ curl -g -X GET http://localhost:6385//v1/nodes '-H Accept: application/json -H Content-Type: application/json -H User-Agent: wait-for-json -H X-Auth-Token: fake-token'
+ jq .
+ echo ' Success!'
 Success!
+ return 0
```

Reply:
> Not specific to this patch but I noticed the wait_for_ironic thing doesn't work.

Ya, I also noticed this wasn't working, I havn't look into why, 

> I noticed this because the default makefile target is still missing ironic, so the containers weren't ruinning:
Opps I thought I added it, will fix now.